presto-tests/src/test/java/com/facebook/presto/tests/TestLocalQueries.java

```diff
@@ -142,4 +167,19 @@ public void testIOExplain()
                 jsonCodec(IOPlan.class).fromJson((String) getOnlyElement(result.getOnlyColumnAsSet())),
                 new IOPlan(ImmutableSet.of(input), Optional.empty()));
     }
+
```

Comment:
Also add a test with the higher limit like 500k or so and make sure it generates the right plan